.ci/main-trigger/Jenkinsfile

```diff
@@ -1,6 +1,6 @@
 // https://github.com/camunda/jenkins-global-shared-library
 // https://github.com/camunda/cambpm-jenkins-shared-library
-@Library(['camunda-ci', 'cambpm-jenkins-shared-library']) _
+@Library(['camunda-ci', 'cambpm-jenkins-shared-library@postgresql-17-declarative-pod-specs']) _
```

Comment:
**When**: Testing is done
**Action**: Remove branch
**Link:** Associated [PR](https://github.com/camunda/camunda-bpm-platform/pull/4552)

Reply:
[Revert commit](https://github.com/camunda/camunda-bpm-platform/pull/4552/commits/0dcea6a602d2722939a2aa5e27948d19693afbdd)